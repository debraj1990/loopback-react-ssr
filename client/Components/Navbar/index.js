import React from 'react';

const Navbar = () => (
    <nav className="navbar navbar-inverse">
    <div className="container-fluid">
      <div className="navbar-header">
        <a className="navbar-brand" href="#">SkillNav</a>
      </div>
      <ul className="nav navbar-nav navbar-right">
        <li><a href="#"><span className="glyphicon glyphicon-user"></span> Sign Up</a></li>
        <li><a href="#"><span className="glyphicon glyphicon-log-in"></span> Login</a></li>
      </ul>
    </div>
  </nav>
)

export default Navbar