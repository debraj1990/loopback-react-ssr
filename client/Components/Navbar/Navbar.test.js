import React from 'react';
import {configure, shallow } from 'enzyme';
import { Link  } from 'react-router-dom'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';


import Navbar from './'


configure({adapter: new Adapter });


describe('Navbar Component', () => {
  
  it('should render the compoent without crushing', () => {
    const component = shallow(<Navbar/>)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  });    
})