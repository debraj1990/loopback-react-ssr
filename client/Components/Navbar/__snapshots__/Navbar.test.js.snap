// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component component should render the compoent without crushing 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container-fluid"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="#"
      >
        SkillNav
      </a>
    </div>
    <ul
      className="nav navbar-nav navbar-right"
    >
      <li>
        <Link
          replace={false}
          to="/signup"
        >
          <span
            className="glyphicon glyphicon-user"
          />
           Sign Up
        </Link>
      </li>
      <li>
        <Link
          replace={false}
          to="/login"
        >
          <span
            className="glyphicon glyphicon-log-in"
          />
           Login
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;

exports[`Navbar Component should render the compoent without crushing 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container-fluid"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="#"
      >
        SkillNav
      </a>
    </div>
    <ul
      className="nav navbar-nav navbar-right"
    >
      <li>
        <Link
          replace={false}
          to="/signup"
        >
          <span
            className="glyphicon glyphicon-user"
          />
           Sign Up
        </Link>
      </li>
      <li>
        <Link
          replace={false}
          to="/login"
        >
          <span
            className="glyphicon glyphicon-log-in"
          />
           Login
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;
